wherescape/wherescape.py

```diff
@@ -71,6 +71,9 @@ def __init__(self):
         self.source_user = os.getenv("WSL_SRCCFG_USER")
         self.source_apikey = os.getenv("WSL_SRCCFG_APIKEY")
 
+        # # source environment
+        # self.source_environment = os.getenv("WSL_SRC_DSN")
+
```

Comment:
1. not sure why commented code is needed in de code base
2. not sure if WSL_SRC_DSN is actually containing the source_environment